+dj/Relvar.m

```diff
@@ -249,6 +249,10 @@ function insert(self, tuples, command)
                             blobs{end+1} = char(v);  %#ok<AGROW>
                         end
                     elseif header.attributes(i).isBlob
+                        assert(~issparse(v), ...
+                            'DataJoint:DataType:Mismatch', ...
+                            'The field `%s` must not be a sparse blob', ...
```

Comment:
Would be better to give message more like "Sparce blob in BLOB is currently not supported"